google-cloud-spanner/src/test/java/com/google/cloud/spanner/AbstractReadContextTest.java

```diff
@@ -110,4 +110,33 @@ public void executeSqlRequestBuilderWithQueryOptions() {
     assertThat(request.getSql()).isEqualTo("SELECT FOO FROM BAR");
     assertThat(request.getQueryOptions().getOptimizerVersion()).isEqualTo("2.0");
   }
+
+  @Test
+  public void executeSqlRequestBuilderWithRequestOptions() {
```

Comment:
Add the `getTransactionTag()` method to the `TestReadContext` test class to simulate a transaction with/without a transaction tag.

Reply:
Done.